REFERENCE.md

```diff
@@ -145,26 +179,92 @@ Apache server's or a virtual host's listening address and port.
 * `apache::peruser::processor`: Enables the `Peruser` module for FreeBSD only.
 * `apache::security::rule_link`: Links the activated_rules from `apache::mod::security` to the respective CRS rules on disk.
 
-### Resource types
-
-* [`a2mod`](#a2mod): Manage Apache 2 modules
-
 ### Functions
 
-* [`apache::apache_pw_hash`](#apacheapache_pw_hash): DEPRECATED.  Use the function [`apache::pw_hash`](#apachepw_hash) instead.
-* [`apache::bool2httpd`](#apachebool2httpd): Transform a supposed boolean to On or Off. Passes all other values through.
-* [`apache::pw_hash`](#apachepw_hash): Hashes a password in a format suitable for htpasswd files read by apache.
-* [`apache_pw_hash`](#apache_pw_hash): DEPRECATED.  Use the namespaced function [`apache::pw_hash`](#apachepw_hash) instead.
-* [`bool2httpd`](#bool2httpd): DEPRECATED.  Use the namespaced function [`apache::bool2httpd`](#apachebool2httpd) instead.
+* [`stdlib::ensure`](#stdlibensure): function to cast ensure parameter to resource specific value
```

Comment:
This looks wrong. As if it created a reference for all fixtures as well.

Reply:
Correct, my fault. Fixed it in the new push